ui/stops/OBAStopViewController.m

```diff
@@ -69,6 +85,7 @@ - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
     
     self.refreshTimer = [NSTimer scheduledTimerWithTimeInterval:kRefreshTimeInterval target:self selector:@selector(reloadData:) userInfo:nil repeats:YES];
+    self.apptentiveTimer = [NSTimer scheduledTimerWithTimeInterval:10 target:self selector:@selector(recordUserVisit:) userInfo:nil repeats:NO];
```

Comment:
True! But what if the user exits the view controller in the intervening time? I only want to display a message to the user if they are still on the stop controller. This raises the very good point that I should probably document this :)